plugins/woocommerce-blocks/assets/js/base/components/price-slider/style.scss

```diff
@@ -310,13 +310,6 @@
 	}
 }
 
-
-/* IE 11 will not support multi-range slider due to poor pointer-events support on the thumb. Reverts to 2 sliders. */
-
-@include ie11() {
-	@include ie-fixes();
```

Comment:
We should be able to remove the `ie-fixes()` mixin from this file.

Reply:
I saw this too however on line 313, it has a comment about Edge browser so I didn't remove it due to that.

Reply:
Oh, right, I missed that one! In any case, it makes reference to Edge 18 and we are currently at [version 126](https://en.wikipedia.org/wiki/Microsoft_Edge). WooCommerce follows the same [browser support policy](https://make.wordpress.org/core/handbook/best-practices/browser-support/) than WordPress, so we don't need to support it either.